dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -729,4 +739,52 @@ public final Response reorderAction(@Context final HttpServletRequest request,
 
         return response;
     } // reorderAction
+
+
+    /**
+     * Returns a set of actions associated to the schemeId
+     * @param request  HttpServletRequest
+     * @param schemeId String
+     * @return Response
+     */
+    @GET
+    @Path("/schemes/{schemeId}/export")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response exportScheme(@Context final HttpServletRequest request,
+                                              @PathParam("schemeId") final String schemeId) {
+
+        final InitDataObject initDataObject = this.webResource.init
+                (null, true, request, true, null);
+        Response response;
+        WorkflowSchemeImportExportObject exportObject;
+        WorkflowScheme                   scheme;
+        Locale                           locale;
+
+        try {
+
+            Logger.debug(this, "Exporting the workflow scheme: " + schemeId);
+            scheme       = this.workflowAPI.findScheme(schemeId);
+            exportObject = this.workflowImportExportUtil.buildExportObject(Arrays.asList(scheme));
+            response     = Response.ok(new ResponseEntityView(exportObject)).build(); // 200
+        } catch (DoesNotExistException e) {
+
+            Logger.error(this.getClass(),
+                    "The Scheme does not exist, id: " + schemeId, e);
+            locale   = LocaleUtil.getLocale(request);
+            response = this.responseUtil.getErrorResponse(request, Response.Status.NOT_FOUND,
+                    locale, initDataObject.getUser().getUserId(), "Workflow-does-not-exists-scheme-id", schemeId);
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)